src/UnixStamp.cpp

```diff
@@ -1,23 +1,10 @@
 #include "UnixStamp.hpp"
 
-UnixStamp::UnixStamp(unixstamp inintialUnix) : UnixStamp(inintialUnix, 0) {}
-
-UnixStamp::UnixStamp(unixstamp inintialUnix, int8_t inintialTz)
-{
-    this->tz = inintialTz;
-    this->unix = inintialUnix - (inintialTz * ONE_HOUR_IN_SEC);
-    this->time = convertUnixToTime(inintialUnix, inintialTz);
-}
-
-UnixStamp::UnixStamp(civil_time initialTime) : UnixStamp(initialTime, 0) {}
+UnixStamp::UnixStamp(unixstamp unix) : unix(unix) {}
 
+UnixStamp::UnixStamp(civil_time time) : UnixStamp(time, 0) {}
 
-UnixStamp::UnixStamp(civil_time initialTime, int8_t initialTz)
-{
-    this->tz = initialTz;
-    this->unix = convertTimeToUnix(initialTime, initialTz);
-    this->time = initialTz != 0 ? convertUnixToTime(this->unix, 0) : initialTime;
-}
+UnixStamp::UnixStamp(civil_time time, int8_t tz) : unix(convertTimeToUnix(time, tz)) {}
```

Comment:
Notice the use of "initializer list" in the constructors (see [Constructors and member initializer lists](https://en.cppreference.com/w/cpp/language/constructor)).